noggin/controller/registration.py

```diff
@@ -355,6 +355,7 @@ def sign_agreements(ipa):
         user=user,
         agreementslist=agreements,
         all_signed=all(agreement.name in user.agreements for agreement in agreements),
+        agreement_warnings=current_app.config.get('AGREEMENT_WARNINGS', {}),
```

Comment:
Since you've added the key to `defaults.py`, you know it's there, you can use:
```suggestion
        agreement_warnings=current_app.config["AGREEMENT_WARNINGS"],
```